cli/src/test/java/com/liferay/blade/cli/ExtensionsTest.java

```diff
@@ -129,7 +129,7 @@ private void _setupTestExtensions() throws Exception {
 		_setupTestExtension(extensionsPath, System.getProperty("sampleTemplateJarFile"));
 	}
 
-	private static final int _NUM_BUILTIN_COMMANDS = 17;
+	private static final int _NUM_BUILTIN_COMMANDS = 18;
```

Comment:
Maybe we can make this variable be a dynamic count of all the lines in this file? `cli/src/main/resources/META-INF/services/com.liferay.blade.cli.command.BaseCommand`